test/functional/zmq_test.py

```diff
@@ -109,6 +114,10 @@ def _zmq_test(self):
         payment_txid = self.nodes[1].sendtoaddress(self.nodes[0].getnewaddress(), 1.0)
         self.sync_all()
 
+        # Should receive the json broadcasted raw transaction.
+        decodedtx = self.decodedtx.receive()
+        assert_equal(payment_txid, json.loads(decodedtx[3:])['txid'])
```

Comment:
Care to explain the offset?

Reply:
Looks like it was some kind of header added by `CDataStream` - I'm not entirely sure, tbh. I switched it over to a vanilla string and char array and the offset went away.